build/lib/Legobot/__init__.py

```diff
@@ -0,0 +1,313 @@
+import socket
```

Comment:
It looks like you rolled this with pip and that generated a `build/lib/...` directory. 

Can you drop these stuff in there from the commit and then add `build/lib/*` to gitignore in a separate PR?